src/services/elasticSearchDomain/connections.ts

```diff
@@ -20,9 +26,14 @@ export default ({
     DomainId,
     VPCOptions: { SecurityGroupIds = [] } = {},
     EncryptionAtRestOptions: { KmsKeyId } = {},
+    CognitoOptions: { IdentityPoolId, UserPoolId, RoleArn } = {},
+    LogPublishingOptions = {},
   } = domain
   const connections: ServiceConnection[] = []
-
+  const logGroupsArns: string[] = Object.entries(LogPublishingOptions).map(
+    ([, value]) => value.CloudWatchLogsLogGroupArn
```

Comment:
could value be undefined here?

Reply:
No, an undefined value is something that cannot be reproduced and wouldn't make much sense to store. The object normally contains a map in which keys are always uppercase with a meaningful string that describes the purpose and the value contains the Cloudwatch log group name and ARN.